cli/flox/src/utils/search.rs

```diff
@@ -233,12 +253,21 @@ impl Display for DisplaySearchResults {
 
         while let Some(d) = items.next() {
             let desc = d.description.as_deref().unwrap_or(DEFAULT_DESCRIPTION);
-            write!(f, "{d:<column_width$}  {desc}", d = d.to_string())?;
+
+            let name = if self.bold_search_term {
+                bold_matches(&d.to_string(), &self.search_term)
+            } else {
+                d.to_string()
+            };
+
+            write!(f, "{name:<column_width$} {desc}", name = name)?;
```

Comment:
```suggestion
            write!(f, "{name:<column_width$}  {desc}", name = name)?;
```

We had two spaces here on purpose because it provides a little more visual breathing room. We should probably leave a comment above the `write!` macro so that other contributors don't think it's an accident.

Reply:
Yes a comment would be nice, will do.